js/fc.js

```diff
@@ -1113,106 +1113,111 @@ var FC = {
                 output: "raw"
             },
             18: {
+                name: "MOD",
```

Comment:
Please add it a LAST entry not to break compatibility with existing solutions

Reply:
I just moved the mod to the last entry.